packages/scss/demo/components/forms/forms-framed.html

```diff
@@ -0,0 +1,186 @@
+<section class="contentSection">
+	<h2>Form mod-framed</h2>
+	<p>
+		Form has can have a mod called <code class="code">.mod-framed</code>.
+		In this mod <code class="code">.form-group</code> behave differently and you have access to a new
+		DOM level called <code class="code">.form-group-line</code>.
+	</p>
+	<p>
+		<code class="code">form-group</code> represents a whole section of your form.
+		It can have a <code class="code">.form-group-title</code> to name your section
+		It can contain <code class="code">.form-group-line</code> and <code class="code">.input.mod-framed</code>
+	</p>
+	<p>
+		In a <code class="code">form-group-line</code> you can use <code class="code">.grid-*</code> classes onto your <code class="code">.input</code> to size
+		your inputs on a single line.
+	</p>
+	<code class="code mod-block">&lt;form class="form mod-framed"&gt;
+	&lt;div class="form-group"&gt;
+		&lt;h2 class="form-group-title"&gt;Title name&lt;/h2&gt;
+		&lt;div class="form-group-line"&gt;
+			&lt;div class="input mod-framed grid-xs8&gt;
+				[...]
+			&lt;/div&gt;
+			&lt;div class="input mod-framed grid-xs4&gt;
+				[...]
+			&lt;/div&gt;
+		&lt;/div&gt;
+	&lt;/div&gt;
+&lt;/form&gt;</code>
+<h4>Example</h4>
+	<div class="form mod-framed">
+		<div class="form-group">
+			<h2 class="form-group-title">Title Group 1</h2>	
+			<div class="form-group-line">
+				<div class="input mod-framed is-required grid-xs8">
+					<input class="input-field is-error" type="text" placeholder="placeholder">
+					<label class="input-label">Input label</label>
+				</div>
+				<div class="input mod-framed grid-xs4">
+					<input class="input-field" type="text" placeholder="placeholder" value="Test disabled" disabled>
+					<label class="input-label">Input label</label>
+				</div>
+			</div>
+			<div class="input mod-framed palette-success">
+				<input class="input-field" type="text" placeholder="placeholder">
+				<label class="input-label">Input label</label>
+			</div>
+			<div class="input mod-framed has-suffix">
+				<input class="input-field" type="text" placeholder="Money Money">
+				<label class="input-label">Input label</label>
+				<div class="input-suffix">
+					€
+				</div>
+			</div>
+			<div class="input textarea mod-framed has-suffix">
```

Comment:
input textarea < Double component?

Reply:
Yes, it avoids style duplication between input.mod-framed and textarea.mod-framed. It may lead to a transformation of textarea as a mod-textarea of input later